### ISSUE TITLE
Warning "objed--do-objects: Wrong type argument: number-or-marker-p, nil" show up after executing objed-delete

### ISSUE THREAD

**Issue by songsanli** [clemera/objed#75]
Thanks for your awesome work!

I see a warning message showing up after I mark multi-lines with <kbd>m</kbd> and press <kbd>d</kbd>:

```
objed--do-objects: Wrong type argument: number-or-marker-p, nil
```

Line objects are deleted successfully but I don't understand why such a message shows up.

**Comment by clemera** [comment]
Thanks for reporting, should be fixed by f5ddab2e5e6a6289bcdc766d48a116dc67619985

**Comment by songsanli** [comment]
Thanks for fixing!
